tests/Rector/Namespace_/FactoryDefinitionRectorConfiguredNoMatch/Fixture/without_namespace.php.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use App;
+
+$factory->define(RectorLaravel\Tests\Rector\Namespace_\FactoryDefinitionRector\Source\Model::class, function ($faker) {
+    return [
+        'name' => $faker->name,
+        'email' => $faker->unique()->safeEmail,
+    ];
+});
+
+?>
+-----
+<?php
+
+use App;
+
+$factory->define(RectorLaravel\Tests\Rector\Namespace_\FactoryDefinitionRector\Source\Model::class, function ($faker) {
+    return [
+        'name' => $faker->name,
+        'email' => $faker->unique()->safeEmail,
+    ];
+});
+
+?>
```

Comment:
This can be removed as a fixture file can just contain the non changing code and it will act as a test to prove that nothing have been changed